### PR TITLE
add third blog post

### PR DESCRIPTION
**Pull request by roseritter**
please merge

### CODE REVIEW

File: source/_posts/2016-10-12-third trial.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Third trial"
+date: "2016-10-12 07:11:21"
+categories: tech-stories
+authors: ["Rose"]
+---
+At [ePages](http://www.epages.com/us/) we provide and constantly improve our existing cloud-based online shop software and therefore count with outstanding software developers and of course a CTO who is dedicated to xxx.
+
+**Rose: What key skills are necessary for a developer to successfully contribute to the ePages shop software?**
```
Comment (erikmueller) [inline source/_posts/2016-10-12-third trial.md:10]: @roseritter think I read that before. Isnt that the same as the first one? Could you provide other content?

Comment (roseritter) [reply source/_posts/2016-10-12-third trial.md:10]: @erikmueller i just tried to get familiar with the creation of pull requests and didn't pay too much attention to content..

Comment (erikmueller) [reply source/_posts/2016-10-12-third trial.md:10]: I know, but I want you to change sth, amend the changes and push to this pull request again 😃 
Don't really care about content either xD

Comment (roseritter) [reply source/_posts/2016-10-12-third trial.md:10]: ahh ok :D sure I'll change it

**Pull request closed without merging**